plots/01_heliostat_position.py

```diff
@@ -34,7 +34,7 @@ class HeliostatPositionPlot:
     Methods
     -------
     load_data()
-        Load the data required for the plot.
+        Load the preprocessing required for the plot.
     get_colour()
```

Comment:
```suggestion
    get_color()
```
I suggest to rename this function to American English `get_color`.